### PR TITLE
release notes wix-window-frontend new value

### PR DESCRIPTION
**Pull request by wix-docs-feds**

### CODE REVIEW

File: guides/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### Updated API: Wix Window Frontend
+
+`wix-window-frontend`
+
+We've updated the [`wix-window-frontend`](https://www.wix.com/velo/reference/wix-window-frontend) module, which contains functionality pertaining to the current browser window:
```
Comment (rebeccask) [inline guides/release-notes.md:14]: you don't need to explain the module. What you DO need to explain is the update. I would do something like this instead:

We've added a new mode, `"Editor"` to the [`viewMode`](https://www.wix.com/velo/reference/wix-window-frontend/viewmode) property of the [`wix-window-frontend`](https://www.wix.com/velo/reference/wix-window-frontend) module.  The `"Editor"` mode is when a visitor is viewing a [Wix Blocks](https://support.wix.com/en/article/wix-blocks-check-out-our-articles) built widget in the editor.

File: guides/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ We love hearing your feedback! You can help us improve the docs by reporting any
...
+- Added, a new value, `"Editor"`, to the [`viewMode`](https://www.wix.com/velo/reference/wix-window-frontend/viewmode) property.
```
Comment (rebeccask) [inline guides/release-notes.md:15]: Also, you don't need a bullet point if it's just one item.

**Review by rebeccask — APPROVED** [review]
Approved with suggested changes.

**Comment by ryangec** [comment]
#skipreview

**Review by rebeccask — APPROVED** [review]

**Comment by ryangec** [comment]
#skipreview

**Pull request closed without merging**